.semaphore/release.yml

```diff
@@ -0,0 +1,39 @@
+version: v1.0
+name: Paprika official release
+agent:
+  machine:
+    type: e1-standard-4
+    os_image: ubuntu1804
+blocks:
+  - name: "🍳 Release"
+    task:
+      secrets:
+        - name: github-ssh
+        - name: paprika-env
+      prologue:
+        commands:
+          - checkout
+          - cache restore client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum yarn.lock),client-node-modules-$SEMAPHORE_GIT_BRANCH,client-node-modules-master
+          - chmod 0600 ~/keys/github.key
+          - ssh-add ~/keys/github.key
+          - git config user.name "Paprika Semaphore 2.0"
+          - git config user.email "allison.chen@wegalvanize.com"
+          - echo "//registry.npmjs.org/:_authToken=$NPM_PUBLISH_TOKEN" >> .npmrc
+      jobs:
+        - name: "Publish to NPM"
+          commands:
+            - git checkout -b release-$SEMAPHORE_PIPELINE_ID
+            - git push origin release-$SEMAPHORE_PIPELINE_ID --no-verify
+            - nvm use
+            - npx lerna bootstrap
+            - yarn
+            - yarn lerna version --conventional-commits --conventional-graduate --no-changelog --no-push --yes
```

Comment:
`--conventional-graduate` will make the official release based on the alpha verison. e.g. `v2.0.1-alpha.12` → `v2.0.1`